website/blog/2023-02-28-ferretdb-release-0-9-2.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "FerretDB v0.9.2 - Tigris Authentication support"
+slug: ferretdb-v-0-9-2
+authors: [alex]
+description: "We are delighted to announce the release of FerretDB version 0.9.2, which includes Tigris authentation, support for `$listIndexes`, `$addtoSet`, and more."
+image: /img/blog/ferretdb_v0.9.2.jpg
+tags: [release]
+date: 2023-02-28
+---
+
+We are delighted to announce the release of FerretDB version 0.9.2, which includes Tigris authentation, support for `$listIndexes`, `$addtoSet`, and more.
```

Comment:
```suggestion
We are delighted to announce the release of FerretDB version 0.9.2, which includes Tigris authentation, support for `listIndexes`, `$addtoSet`, and more.
```

`listIndexes` is a command: https://www.mongodb.com/docs/manual/reference/command/listIndexes/
`$addtoSet` is an operator: https://www.mongodb.com/docs/manual/reference/operator/update/addToSet/